lib/check-translation.js

```diff
@@ -246,10 +290,16 @@ CheckTranslation.prototype = {
     } catch (err) {
       this._finish(err);
     }
-  }
+    this._lastTagName = tagName;
```

Comment:
I cannot find where this assigned value is used.

Reply:
Ahha! That's because it's not!